### PR TITLE
Add Ask Ubuntu

### LINKED ISSUE #5095: Request: Ask Ubuntu icon
**Brand Name:** Ask Ubuntu
**Website:** https://askubuntu.com/
**Alexa rank:** [295k](https://www.alexa.com/siteinfo/askubuntu.com)

**Official resources for icon and color:**
I couldn't find an official media kit, but in the landing page source code there's [this svg](https://cdn.sstatic.net/Sites/askubuntu/Img/logo.svg) that could be easily cropped to fit only the "Ask" logo.
About the official color, I'm not really sure, as I've found the following versions:
- [`#dd4c00`](https://cdn.sstatic.net/Sites/askubuntu/Img/apple-touch-icon.png) 
- [`#f47323`](https://cdn.sstatic.net/Sites/askubuntu/Img/apple-touch-icon@2.png) 

- `#dc461d` in the [logo's topbar](https://askubuntu.com)

I could work on this, but I'd like opinions about what you guys do for these cases where doesn't seem to be defined an official color.

### PR DESCRIPTION
**Pull request by alagos**
askubuntu

**Issue:** Closes #5095 
**Alexa rank:** [~3012](https://www.alexa.com/siteinfo/askubuntu.com)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
No media kit found, but I used [this svg](https://cdn.sstatic.net/Sites/askubuntu/Img/logo.svg) as base (Logo located in the header of [askubuntu.com](https://askubuntu.com/)), then I cropped it to fit only the "ask" logo.
`#dc461d` used, as it was suggested [in the original ticket](https://github.com/simple-icons/simple-icons/issues/5095#issuecomment-785834263).

### CODE REVIEW

**Comment by PeterShaggyNoble** [comment]
Welcome to Simple Icons, @alagos and thanks for taking this one on :+1:

Everything looks pretty good to me. However, before we merge it, could you see if you could clean up some of the points in the lettering?

**Comment by alagos** [comment]
@PeterShaggyNoble with that you meant to tick all the checklist? because I already optimized the file, I just forgot to tick it (now is done)

File: _data/simple-icons.json
Comment (mondeja) [inline _data/simple-icons.json:767]: Could you add the https://stackoverflow.com/legal/trademark-guidance as the `guidelines` entry as it was added for the rest of the StackExchange icons in #5164?

```suggestion
            "source": "https://askubuntu.com/",
            "guidelines": "https://stackoverflow.com/legal/trademark-guidance"
```

**Review by mondeja — CHANGES_REQUESTED** [review]
Thanks for submit this pull @alagos 👍🏻

**Comment by PeterShaggyNoble** [comment]
Sorry, @alagos, should have been clearer on that: the letters in the source file consist of more points than are necessary (see below), which we should try to clean up if at all possible. This is as a result of them originally having been a `text` element that was later converted to a `path`. I'm unsure, though, if it's a software specific thing or just particular to Inkscape so, if you're using something else, it may be easier to install [this font](https://www.gnome.org/fonts/) and use the SVG @service-paradis found [here](https://stackapps.com/questions/347/svg-versions-of-the-logos) as a starting point, rather than trying to fix the version from their website manually.

**Comment by PeterShaggyNoble** [comment]
Took a run at cleaning this up myself but it turns out that Inkscape's "Simplify path" tool actually means "Butcher it mercilessly" and trying to do it manually, before or after simplification, is just insanity!

So, unless someone else can do a better job or recreate it from scratch, I say we merge this in as-is for now.

**Comment by alagos** [comment]
@mondeja guidelines added

**Comment by PeterShaggyNoble** [comment]
Thanks, @alagos. As quite some time has passed with nobody offering an improvement with regard to the number of points in the lettering, I'm going to merge this in as it is and if anyone can clean it up in the future then we can address that in a separate PR.

**Pull request merged**
